cli/pcluster/easyconfig.py

```diff
@@ -290,3 +303,14 @@ def _is_config_valid(args, config):
         if is_file_ok:
             return True
         return False
+
+
+def _is_aws_batch(scheduler):
```

Comment:
the name should be `_is_awsbatch`, in any case I think we can avoid this function since it is used in a single place.